system/Cookie/Cookie.php

```diff
@@ -445,7 +401,7 @@ public function getOptions(): array
 			'domain'   => $this->domain,
 			'secure'   => $this->secure,
 			'httponly' => $this->httponly,
-			'samesite' => $this->samesite ?: ucfirst(self::SAMESITE_LAX),
+			'samesite' => $this->samesite,
```

Comment:
Please revert. This is for fallback catch for the blank samesite setting.

Reply:
On `setrawcookie()` and `setcookie()` calls, empty samesites would cause the cookie to be rejected for creation.